### PR TITLE
add jbake and markdown

### PR DESCRIPTION
**Pull request by ghaseminya**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the guidelines. 
        
    ``[Name](http://homepage/) - Short description. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code)) `License` `Language```
- [x] Your additions are ordered alphabetically.
- [x] Your additions are [Free software](https://en.wikipedia.org/wiki/Free_software), or if not they have been added to [non-free](non-free.md).
- [x] Your additions are not already listed at [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin) (IT infrastructure management), [staticgen.com](https://www.staticgen.com/) or [staticsitegenerators.net](https://staticsitegenerators.net/) (static site generators).
- [x] Any licenses you have added are in our [list of licenses](https://github.com/Kickball/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant issues or PRs.
- [x] Any category you are creating has the minimum requirement of 3 items.

### CODE REVIEW

**Comment by Kickball** [comment]
JBake is already listed on https://staticsitegenerators.net/, please remove it from the PR.

**Comment by ghaseminya** [comment]
thanks @Kickball 
I didn't see jbake,sorry. I remove it.

File: README.md [hunk 1]
```diff
@@ -803,7 +803,9 @@ See https://github.com/n1trux/awesome-sysadmin#monitoring and https://github.com
 
 *See also [Wikis](#wikis)*
 
+
   * [dillinger](http://dillinger.io/) - The last Markdown editor, ever. ([Source Code](https://github.com/joemccann/dillinger/)) `MIT` `Nodejs`
+  * [domchristie](http://domchristie.github.io/to-markdown/) - An HTML to Markdown converter written in JavaScript. [Source Code](https://github.com/domchristie/to-markdown) `MIT` `JavaScript`
```
Comment (Kickball) [inline README.md]: Shouldn't this be called to-markdown?

Comment (ghaseminya) [inline README.md]: yes, it must be change to to-markdown

File: README.md [hunk 2]
```diff
@@ -803,6 +803,7 @@ See https://github.com/n1trux/awesome-sysadmin#monitoring and https://github.com
 
 *See also [Wikis](#wikis)*
 
+
```
Comment (nodiscc) [inline README.md]: Please remove the added blank line

File: README.md [hunk 3]
```diff
@@ -1040,7 +1042,8 @@ See also [Static site generators](#static-site-generators)
   * [Flatdoc](http://ricostacruz.com/flatdoc/) - A small Javascript file that fetches Markdown files and renders them as full pages. `MIT` `Javascript`
   * [markdown-tree](https://github.com/mil/markdown-tree) - Serve a hierarchy / tree directory of markdown files. Use intended for small sites built in markdown. `MIT` `Ruby`
   * [Read the Docs](https://docs.readthedocs.org/en/latest/install.html) - Host documentation, making it fully searchable and easy to find; import your docs using any major version control system, including Mercurial, Git, Subversion, and Bazaar. ([Demo](https://readthedocs.org/projects/), [Source Code](https://github.com/rtfd/readthedocs.org)) `MIT` `Python`
-
+  
+
```
Comment (nodiscc) [inline README.md]: Please remove the added blank line

**Comment by nodiscc** [comment]
@ghaseminya Can you rebase on top of the current `master` branch? (there are merge conflicts) (and possibly squash 6 commits into one) Thanks

**Comment by Kickball** [comment]
@nodiscc I think we can squash the commits when we merge it, but I am unable to resolve the merge conflicts. 

@ghaseminya have you made any progress with the conflict resolution?

**Comment by ghaseminya** [comment]
@nodiscc i try to rebase but i couldn't rebase because switch button in pull request page not found.
@Kickball no, because i couldn't refresh my repo and then PR

**Comment by nodiscc** [comment]
I have rebased it and moved to https://github.com/Kickball/awesome-selfhosted/pull/1066

**Pull request closed without merging**